collections.go

```diff
@@ -104,19 +108,51 @@ func (cl *CollectionsListClient) Do(ctx context.Context, fn CollectionsListFn) e
 		}
 
 		// If we are here then it means there was no error/bad response while fetching current page
-		// so lets iterate over page items.
+		// so let's iterate over page items.
 		for _, col := range success.Data {
 			if ok, e := fn(col, nil); !ok {
 				return e
 			}
 		}
 
-		// If there are more than one items in current list then there could be more items remaining to be fetched. In
+		// If there are more than one item in current list then there could be more items remaining to be fetched. In
```

Comment:
Yes, this is partial correction, full correction would be:
`If there is more than one item in current...`